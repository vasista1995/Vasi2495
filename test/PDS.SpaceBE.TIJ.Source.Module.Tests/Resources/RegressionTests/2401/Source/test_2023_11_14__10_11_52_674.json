{
  "Module": "SP",
  "Segment": "Tijuana",
  "FourDReport": "*",
  "ProcessEquipment": "WSDI0109",
  "ParameterClass": "NON_PPAP",
  "Category": "PT-0008-07",
  "Classification": "SPC-Monitoreo",
  "Operator": "omarroq1",
  "PartNumber": "PA002410340",
  "Track": "Z1",
  "Area": "Pre_assy_Corte",
  "SpaceRawValues": [
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096851,
      "Seqnr": 1,
      "Value": 0.0305,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096851,
      "Seqnr": 5,
      "Value": 0.0305,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096851,
      "Seqnr": 4,
      "Value": 0.0305,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096851,
      "Seqnr": 3,
      "Value": 0.0315,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    },
    {
      "BreakPoint": "Rawvalueslist",
      "SampleId": 3096851,
      "Seqnr": 2,
      "Value": 0.0315,
      "ExternalFlagged": "N",
      "InternalFlagged": "N",
      "NumViolations": 0
    }
  ],
  "UniqueSourceKey": "TIJ:BE:104OOG8.1:Pre_assy_Corte:SP:WaferKerf_Width:1834:3096851:0",
  "PKey": "TIJ:BE:104OOG8.1:Pre_assy_Corte:SP:WaferKerf_Width:1834:3096851:0",
  "ParameterName": "WaferKerf_Width",
  "ParameterUnit": "mm",
  "SampleDate": "2023-10-09T19:51:41Z",
  "SampleDateUtc": "2023-10-10T02:51:41Z",
  "SampleSize": 5,
  "Sigma": ",000547722557505167",
  "Mean": ",0309",
  "Median": ",0305",
  "Min": ",0305",
  "Max": ",0315",
  "Q1": ",0305",
  "Q3": ",0315",
  "LdsID": "2401",
  "IsCalcParam": 0,
  "MeanCntrlLow": 0.02812,
  "MeanCntrlTarget": 0.03103,
  "MeanCntrlHigh": 0.03393,
  "RangeCntrlLow": 0.0,
  "RangeCntrlTarget": 0.00127,
  "RangeCntrlHigh": 0.00254,
  "SpecTarget": 0.037,
  "SpecHigh": 0.0508,
  "CtrlMeanHighEnabled": "Y",
  "CtrlMeanLowEnabled": "Y",
  "CtrlMeanTargetEnabled": "Y",
  "CtrlLowEnabled": "N",
  "CtrlHighEnabled": "N",
  "CtrlRangeLowEnabled": "N",
  "CtrlRangeHighEnabled": "N",
  "CtrlSigmaLowEnabled": "Y",
  "CtrlSigmaCenterEnabled": "Y",
  "CtrlRangeCenterEnabled": "N",
  "CtrlCenterEnabled": "N",
  "CtrlSigmaHighEnabled": "Y",
  "SpecLowEnabled": "Y",
  "SpecHighEnabled": "N",
  "CreatedDate": "2023-10-09T19:51:41Z",
  "CreatedDateUtc": "2023-10-10T02:51:41Z",
  "UpdatedDate": "2023-10-09T19:51:41Z",
  "UpdatedDateUtc": "2023-10-10T02:51:41Z",
  "RvStoreFlag": "Y",
  "Lot": "104OOG8.1",
  "ChannelId": "1834",
  "CkcId": "0",
  "ChannelName": "WaferKerf_Width/Pre_assy/PT-0008-07",
  "ChannelDescr": "Wafer saw process",
  "SpaceInstanceName": "BE",
  "Sitekey": "TIJ",
  "CalcCreated": "10/09/2020 18:49:24",
  "ILdsID": 0,
  "AcceptFlag": "Y",
  "UpdatedBy": " ",
  "SpecTargetOrigin": "V",
  "ViolCount": 0,
  "ExtMaMV": 0.03056,
  "ExtEwmaMV": 0.030660383289931,
  "ExtMsMV": 0.000141421356237311,
  "OutOfOrder": "N",
  "EwmaS": 0.000555795729412499,
  "EwmaR": 0.001181444608,
  "Calc2ID": 64863,
  "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN",
  "ExtEwmaMVLCL": 0.0,
  "ExtEwmaMVCenter": 0.0,
  "ExtEwmaMVUCL": 0.0,
  "ExtMaMVLCL": 0.0,
  "ExtMaMVCenter": 0.0,
  "ExtMaMVUCL": 0.0,
  "EwmaSLCL": 0.0,
  "EwmaSCenter": 0.0,
  "EwmaSUCL": 0.0,
  "EwmaRLCL": 0.0,
  "EwmaRCenter": 0.0,
  "EwmaRUCL": 0.0,
  "ExtMSMVLCL": 0.0,
  "ExtMSMVCenter": 0.0,
  "ExtMSMVUCL": 0.0,
  "ClOrigin": 0,
  "MVLal": 0.0,
  "MVUal": 0.0,
  "RawLal": 0.0,
  "RawUal": 0.0,
  "SigmaLal": 0.0,
  "SigmaUal": 0.0,
  "RangeLal": 0.0,
  "RangeUal": 0.0,
  "ExtSpecLower": 0.0,
  "ExtSpecUpper": 0.0,
  "ExtSpecLimEnable": "NN",
  "CalculationStrategy": "ND",
  "ChState": "Online"
}