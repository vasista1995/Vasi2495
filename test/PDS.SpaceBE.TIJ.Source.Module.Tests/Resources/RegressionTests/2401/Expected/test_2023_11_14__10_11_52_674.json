{
  "IdSource": "SPACEACT2:TIJ:BE:104OOG8.1:Pre_assy_Corte:SP:WaferKerf_Width:1834:3096851:0",
  "Data_flat_Metadata": {
    "Segment": "Tijuana",
    "Module": "SP",
    "FourDReport": "*",
    "ParameterClass": "NON_PPAP",
    "ProcessEquipment": "WSDI0109",
    "Category": "PT-0008-07",
    "Classification": "SPC-Monitoreo",
    "Operator": "omarroq1",
    "PartNumber": "PA002410340",
    "Track": "Z1",
    "Area": "Pre_assy_Corte",
    "Lot": "104OOG8.1",
    "ParameterName": "WaferKerf_Width",
    "ParameterUnit": "mm",
    "SiteKey": "TIJ",
    "ChannelId": "1834",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-10T02:51:41Z",
    "CreatedTimestamp": "2023-10-09T19:51:41Z",
    "UpdatedTimestamp": "2023-10-09T19:51:41Z",
    "UpdatedTimestampUtc": "2023-10-10T02:51:41Z",
    "ExportedTimestampUtc": "2023-10-09T19:51:41Z",
    "ExportedTimestamp": "2023-10-09T19:51:41Z",
    "SampleTimestampUtc": "2023-10-10T02:51:41Z",
    "SampleTimestamp": "2023-10-09T19:51:41Z",
    "SourceDataLevel": "C",
    "ChannelName": "WaferKerf_Width/Pre_assy/PT-0008-07",
    "ChannelDescr": "Wafer saw process",
    "RvStoreFlag": "Y",
    "CalcCreated": "10/09/2020 18:49:24",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 64863,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2401",
    "IsCalcParam": 0,
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "N",
        "SpecLowEnabled": "Y",
        "SpecHigh": 0.0508,
        "SpecTarget": 0.037,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": ",0309",
        "Median": ",0305",
        "Sigma": ",000547722557505167",
        "Min": ",0305",
        "Max": ",0315",
        "Q1": ",0305",
        "Q3": ",0315"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.02812,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 0.03103,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.03393,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 0.0,
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.00127,
        "ctrl_range_high": 0.00254,
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 0.03056,
        "ExtEwmaMV": 0.030660383289931,
        "ExtMsMV": 0.000141421356237311,
        "EwmaS": 0.000555795729412499,
        "EwmaR": 0.001181444608,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 3096851,
        "Seqnr": 1,
        "Value": 0.0305,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096851,
        "Seqnr": 5,
        "Value": 0.0305,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096851,
        "Seqnr": 4,
        "Value": 0.0305,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096851,
        "Seqnr": 3,
        "Value": 0.0315,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096851,
        "Seqnr": 2,
        "Value": 0.0315,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:TIJ:BE:Pre_assy_Corte:SP:WaferKerf_Width:1834",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:104OOG8.1",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "TIJ:BE:104OOG8.1:Pre_assy_Corte:SP:WaferKerf_Width:1834:3096851:0"
  }
}